src/components/summary-list/index.md.njk

```diff
@@ -34,6 +34,11 @@ There are 2 ways to use the summary list component. You can use HTML or, if you
 
 You can add actions to a summary list, like a ‘Change’ link to let users go back and edit their answer.
 
+The actions get their context from the content in their row.
```

Comment:
This looks pretty good to me, nice work.

I'm suggesting a couple of wording changes to reduce the repetition of the word "context" and, ironically, to provide a little more context. As follows:

"For sighted users, the actions get their context from the other content in the row they appear in. 

Assistive technology users, like those who use a screen reader, may not get this context if they read the links in isolation. To give the links more context, add visually hidden text to the link. This means a screen reader will announce a meaningful action, like ‘Change name’ or ‘Change date of birth’."

Reply:
'out of context' is the phrase that is normally used rather than 'in isolation'. But I agree there are a lot of contexts.

I agree with @nickcolley that 'hear' is weird but 'announce' is not as plain as it could be.

How about:

"For sighted users, the actions get their context from the other content in the row they appear in.

"Assistive technology users, like those who use a screen reader, may read the links out of context and not know what the link does. To give the links more context, add visually hidden text to the link. This means a user will hear a meaningful action, like ‘Change name’ or ‘Change date of birth’."

Reply:
@amyhupe @stevenaproctor that's looking really good, I'll update with Steven's update to remove announce as I agree that's somewhat a technical term.